src/ONYXKEYS.ts

```diff
@@ -63,6 +63,9 @@ const ONYXKEYS = {
     /** Contains all the info for Tasks */
     TASK: 'task',
 
+    /** Contains all the info for Workspace Rate and Unit (editing) */
+    WORKSPACE_RATE_AND_UNIT: 'workspaceRateAndUnit',
```

Comment:
Is the intention of this to simply store form values while rate and unit are being configured? If so, can you please update the comment to have a little bit more of that context? I also believe that you can only edit one workspace at a time, so it would be good to mention that in the comment as well. This is using a little bit of a different pattern than most forms and the first two questions I had when I saw this was:

1. Why isn't this part of `ONYXKEYS.COLLECTION.POLICY` (maybe like `WORKSPACE_INVITE_MEMBERS_DRAFT` for example)?
2. Why isn't this part of `ONYXKEYS.FORM.<something>`?

Reply:
Thanks, that's all good and I think it's OK how you have it. Can you please add this context to the code comment?